src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2566,7 +2560,11 @@ namespace NServiceBus.Settings
     public class SettingsHolder : NServiceBus.Settings.ReadOnlySettings
     {
         public SettingsHolder() { }
+        [System.ObsoleteAttribute("Property injection is no longer supported, please use constructor injection inste" +
+            "ad. Will be removed in version 7.0.0.", true)]
```

Comment:
I am using obsoleteex, this is just the apifile we generate in the approval tests we use.

This is the current usage, https://github.com/Particular/NServiceBus/blob/obsolete-set-property/src/NServiceBus.Core/Settings/SettingsHolder.cs#L181, can we leave this as is?